### DISCUSSION TITLE
How to sync the Git source to the latest release and to get symbol files

### DISCUSSION THREAD

**Discussion by JohnTasler** [unoplatform/uno#12798] [Q&A]
Firstly, I would like to debug into the source code of Uno and other such support libraries (e.g., Uno.Tookit.UI).

My first thought was to sync-up to the same release as the NuGet's I'm using, and build the Debug symbols. However, VS shows the latest NuGet's as 4.9.20 and 4.9.26. The latest release tag on GitHub is 4.9.17. So how can I sync-up to this release?

Also, am I making this harder than it should be? What's the best practice for debugging? Should I not be using NuGets, but use the built assemblies directly?

**Comment by jeromelaban** [comment]
Debuging libraries has always been a [not-so-fun part of the Nuget ecosystem](https://github.com/NuGet/Home/issues/7087). That's why we're using our own "nuget override" system, described [here for Uno](https://platform.uno/docs/articles/uno-development/debugging-uno-ui.html#debugging-uno-in-another-application), and [here for uno.toolkit.ui](https://github.com/unoplatform/uno.toolkit.ui/blob/50c8f182c19f2c0d0ac9874d5b848a48d0342781/src/crosstargeting_override.props.sample#L15).

The latest published package is always the tip of the `release/stable/xx.yy` branch (e.g. https://github.com/unoplatform/uno.toolkit.ui/tree/release/stable/2.5).
